ape_safe/accounts.py

```diff
@@ -47,17 +51,36 @@ def accounts(self) -> Iterator[AccountAPI]:
     def save_account(self, alias: str, address: str):
         """
         Save a new Safe to your ape configuration.
+
+        Args:
+            alias (str): The alias to save the Safe under.
+            address (str): The address of the Safe account.
         """
         chain_id = self.provider.chain_id
         account_data = {"address": address, "deployed_chain_ids": [chain_id]}
         path = self.data_folder.joinpath(f"{alias}.json")
         path.write_text(json.dumps(account_data))
 
     def load_account(self, alias: str) -> "SafeAccount":
+        """
+        Loads the Safe account.
```

Comment:
```suggestion
        Load the Safe account.
```

Doc strs should answer the question "What can I do with this method?"
Here the answer is "Load the Safe account".

so the tense of the verbs is non-conforming for all these